### ISSUE TITLE
improve extension start time by lazy load command module

### ISSUE THREAD

**Issue by tjx666** [qcz/vscode-text-power-tools#58]
You can check here for example: https://github.com/tjx666/vscode-fe-helper/blob/master/src/extension.ts#L26

Although this extension is fast to activate, but still can much improve startup speed

**Comment by qcz** [comment]
Good idea, especially with commands with big dependencies (voca, faker etc.).

Will consider in the future when a new, bigger dependency is required for a feature. Don't know how it affects the browser version though.
